src/workerd/api/queue.c++

```diff
@@ -179,21 +190,30 @@ kj::Promise<void> WorkerQueue::send(jsg::Lock& js,
 
   JSG_REQUIRE(!body.isUndefined(), TypeError, "Message body cannot be undefined");
 
+  auto headers = kj::HttpHeaders(context.getHeaderTable());
+  headers.set(kj::HttpHeaderId::CONTENT_TYPE, MimeType::OCTET_STREAM.toString());
+
   kj::Maybe<kj::StringPtr> contentType;
   KJ_IF_SOME(opts, options) {
     KJ_IF_SOME(type, opts.contentType) {
       contentType = validateContentType(type);
+      headers.add(HDR_MSG_FORMAT, type);
+    }
+    KJ_IF_SOME(secs, opts.delaySecs) {
+      if (secs > MAX_DELIVERY_DELAY_SECS) {
+        JSG_FAIL_REQUIRE(TypeError,
+                         kj::str("The delivery delay must be a value between 0 and ",
+                                 MAX_DELIVERY_DELAY_SECS, " inclusive (current: ", secs, ")."));
+      }
+      headers.add(HDR_MSG_DELAY, kj::str(secs));
     }
   }
 
-  auto headers = kj::HttpHeaders(context.getHeaderTable());
-  headers.set(kj::HttpHeaderId::CONTENT_TYPE, MimeType::OCTET_STREAM.toString());
-
   Serialized serialized;
   KJ_IF_SOME(type, contentType) {
-    headers.add("X-Msg-Fmt", type);
     serialized = serialize(js, body, type, SerializeArrayBufferBehavior::DEEP_COPY);
-  } else {
+  }
+  else {
```

Comment:
We always keep `else` on the same line as the prior closing brace, so I'd suggest getting your auto-formatter (`make format`) working properly :)

Reply:
Huh, that's weird. I don't know much about our clang-format setup since it arose after I did most of my runtime work, but here's what I know:

* `make format` in edgeworker doesn't appear to touch files in workerd
* When I change a line in edgeworker to move the `else` to the following line like this, running `make format` moves it back to the same line
* The style guide we follow is [pretty clear about the intended format here](https://github.com/capnproto/capnproto/blob/v2/style-guide.md#spacing-and-bracing) and all the code I can recall reading in the runtime code follows that style

I don't know much about our clang-format config for workerd, though. Maybe @jasnell has a pointer here, but no worries if you don't.